locale/en_US/submission.xml

```diff
@@ -44,6 +44,7 @@
 	<message key="submission.confirmSubmit">Are you sure you wish to submit this manuscript to the press?</message>
 	<message key="submission.metadata">Metadata</message>
 	<message key="submission.supportingAgencies">Supporting Agencies</message>
+	<message key="submission.grid.titleColumn">Author; Book Title</message.
```

Comment:
@NateWr I think this should go into library locale files, with the "book" word removed, something like Author; Title. Usually we try to make the pkp library an encapsulated component, so if you use a locale key in pkp library, the locale key definition should also be in pkp library. That way it's much easier for us to maintain and use the code.

It it's a requirement to have the word that defines the kind of object (book, article, etc), then I would create a template method that defines the locale key for this to be implemented in the application level and use this template method in the library. Makes sense?